src/fcmcts/fcmcts.jl

```diff
@@ -259,31 +259,31 @@ function compute_best_action_node(mdp::JointMDP, tree::FCMCTSTree, node::FCState
     return best
 end
 
-POMDPLinter.@POMDP_require simulate(planner::FCMCTSPlanner, s, depth::Int64) begin
+@POMDP_require simulate(planner::FCMCTSPlanner, s, depth::Int64) begin
     mdp = planner.mdp
     P = typeof(mdp)
     @assert P <: JointMDP       # req does different thing?
-    SV = statetype(P)
-    @assert typeof(SV) <: AbstractVector # TODO: Is this correct?
+    #SV = statetype(P)
```

Comment:
Uncomment? Since we are using `::SV` further down?

Reply:
But `::SV` need not be `AbstractVector` and can have other global info too?

Reply:
It makes sense to uncomment `SV` since it is used further down